nss-ice-phantoms/BangazonAPI
# Rr bangazon react

# Description

Adds initial setup for the Bangazon React App.

## Type of change

# Testing Instructions

1. `git fetch --all`
2. `git checkout rr-bangazonReact`
3.  start BangazonAPI in Visual Studio
4.  from terminal go to BangazonAPI/BangazonReact/ClientApp and run 'npm start' (you may need to run npm install in this directory first)
5.  Chrome should open and the page should display the names of the 3 departments in the database

# Checklist:
